src/Components/WebAssembly/WebAssembly.Authentication/src/Services/RemoteAuthenticationService.cs

```diff
@@ -74,7 +81,9 @@ public class RemoteAuthenticationService<TRemoteAuthenticationState, TProviderOp
             var result = internalResult.Convert();
             if (result.Status == RemoteAuthenticationStatus.Success)
             {
-                UpdateUser(GetUser());
+                var getUserTask = GetUser();
+                await getUserTask;
+                UpdateUser(getUserTask);
```

Comment:
This seems problematic. `GetUser` returns a `ValueTask<T>` which appears to be [awaited multiple times](https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.valuetask-1?view=netcore-3.1#remarks)

Reply:
Well spotted

Reply:
Thanks, I've changed this to be a Task.